server/routes/journeys/record-review/close-csip/tests.cy.ts

```diff
@@ -0,0 +1,99 @@
+import { v4 as uuidV4 } from 'uuid'
+import { checkAxeAccessibility } from '../../../../../integration_tests/support/accessibilityViolations'
+import { injectJourneyDataAndReload } from '../../../../../integration_tests/utils/e2eTestUtils'
+import { todayString } from '../../../../utils/datetimeUtils'
+
+context('test /record-review/close-csip', () => {
+  const uuid = uuidV4()
+  const START_URL = `${uuid}/csip-record/02e5854f-f7b1-4c56-bec8-69e390eb8550/record-review/start`
+  const PAGE_URL = `${uuid}/record-review/close-csip`
+
+  const getCloseButton = () => cy.findByRole('button', { name: 'Yes, close CSIP' })
+  const getCancelButton = () => cy.findByRole('button', { name: 'No, change the outcome' })
+
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignIn')
+    cy.task('stubComponents')
+    cy.task('stubGetPrisonerImage')
+    cy.task('stubGetPrisoner')
+    cy.task('stubCsipRecordGetSuccess')
+  })
+
+  it('should try out all cases - with no open identified needs', () => {
+    cy.signIn()
+    cy.visit(START_URL, { failOnStatusCode: false })
+    cy.visit(PAGE_URL)
+
+    cy.get('.govuk-grid-column-two-thirds').children().should('have.length', 5)
+
```

Comment:
it's not obvious what is being tested here.
can it instead findBy something else and `should('not.exists')` so we know what you're trying to assert?